xds/src/main/java/io/grpc/xds/ClusterImplLoadBalancer.java

```diff
@@ -236,7 +236,7 @@ public void start(SubchannelStateListener listener) {
           delegate().start(new SubchannelStateListener() {
             @Override
             public void onSubchannelState(ConnectivityStateInfo newState) {
-              if (newState.getState().equals(ConnectivityState.READY)) {
+              if (xdsClient != null && newState.getState().equals(ConnectivityState.READY)) {
```

Comment:
Let's add a comment as to why we are checking for null. Something like, `// Do nothing if LB has been shut down`